chromadb/api/__init__.py

```diff
@@ -118,6 +126,7 @@ def _modify(
         current_name: str,
         new_name: Optional[str] = None,
         new_metadata: Optional[Dict] = None,
+        new_index_params: Optional[HnswIndexParams] = None,
```

Comment:
What does modification mean here? Changing index params would mean rebuilding the whole index. Seems like we should either do that, or error if you try to change the params after the index has already been built.

Reply:
I think for now we just don't allow index modification.